docs/welcome_and_overview.md

```diff
@@ -1,4 +1,78 @@
 # Introduction
 
-Welcome to the documentation repository for Synapse, the reference
-[Matrix](https://matrix.org) homeserver implementation.
\ No newline at end of file
+Welcome to the documentation for Synapse, the reference
+[Matrix](https://matrix.org) homeserver implementation.
```

Comment:
reference my foot.

```suggestion
Welcome to the documentation for Synapse.
```

cf https://github.com/matrix-org/synapse/issues/10675

Reply:
Oh! This may be worth calling out internally, as I thought that was still the official word.